cirq-google/cirq_google/devices/serializable_device_test.py

```diff
@@ -121,7 +121,7 @@ def test_metadata_correct():
         pairs = [
             (qubits[0], qubits[1]),
             (qubits[0], qubits[3]),
-            (qubits[1], qubits[4]),
+            (qubits[4], qubits[1]),
```

Comment:
Sorry, can you describe what _was_ happening here and what _should_ be happening here? Was this a latent bug? What this pair just not added to the device topology because the constructor expects ordered pairs but got unordered pairs? What defense do you have against passing unordered pairs to the GridMetadata constructor?

Reply:
Looked into this again, and turns out the previous code was fine. For each `SYMMETRIC` target in the `DeviceSpecification` proto, two tuples are created in the gate definition, one for each direction of the pair:

https://github.com/quantumlib/Cirq/blob/1769cba0675dadd02e77ccae87be82757c96bb5d/cirq-google/cirq_google/devices/serializable_device.py#L223

so the `pair[0] < pair[1]` deduplicates the pair, and no qubit pairs should be omitted.

I'll go ahead and close this PR (including the test case change), since `SerializableDevice` will be removed soon. But let me know if you feel otherwise.

> What defense do you have against passing unordered pairs to the GridMetadata constructor?
In the current `GridDeviceMetadata` implementation, qubit pairs passed into the constructor are considered to be unordered

Reply:
That works as long as you're coming via that proto code path, but if you're just using the bare `__init__` method, could you get yourself into hot water?

Reply:
Yeah that is true. I am OK either way since the preferred method to construct `SerializableDevice` is `from_proto()` (as mentioned in the `__init__` docstring).

Reply:
that seems dangerous to me.

When is this class getting removed?

Reply:
bump

Reply:
This class is now gone in Cirq 1.0. PR: https://github.com/quantumlib/Cirq/pull/5743